tests/phpunit/CRM/Utils/RuleTest.php

```diff
@@ -84,7 +84,10 @@ public function numericDataProvider() {
    * @param $inputData
    * @param $expectedResult
    */
-  public function testMoney($inputData, $expectedResult) {
+  public function testMoney($inputData, $decimalPoint, $thousandSeparator, $currency, $expectedResult) {
```

Comment:
I think this will cause jenkins to fail - not sure if that is the issue giving the weird test error atm - the params need to be in the comment block (can you also add 'type' to the exisiting ones as we are trying to up our game - ie ```@param array $inputData```